recipes/contact_map/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "contact_map" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "695967007dc43b639efd458130411463ce79893863699d2d32b60c1b848cd062" %}
+# sha256 comes from 
+# openssl sha256 dist/contact_map-{{ version }}.tar.gz
+# (after `python setup.py sdist`)
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
When you use `--single-version-externally-managed`, you only typically need `python` and `setuptools` as `build` requirements (unless `setup.py` imports other modules).